libfreefare/ntag21x.c

```diff
@@ -282,7 +286,7 @@ ntag21x_get_info(FreefareTag tag)
 	NTAG_21x(tag)->subtype = NTAG_216;
 	break;
     default:
-	NTAG_21x(tag)->subtype = NTAG_213;
+	NTAG_21x(tag)->subtype = NTAG_UNKNOWN;
```

Comment:
I am wondering, hasn't `subtype` already been initialized in `_ntag21x_tag_new()`?

Reply:
Yes this line is unnessesary, I will remove it :+1: